bbl_screen-patch/patches/printerui/qml/x1plus/GpioActions.js

```diff
@@ -0,0 +1,192 @@
+.pragma library
+.import X1PlusNative 1.0 as JSX1PlusNative
+.import "Binding.js" as Binding
+
+var X1Plus = null;
+var _X1PlusNative = JSX1PlusNative.X1PlusNative;
+
+
+var settings_file = "/mnt/sdcard/x1plus/buttons.json"
+
+const ACTION_TOGGLE_SCREENSAVER = "Sleep/wake";
+const ACTION_REBOOT = "Reboot";
+const ACTION_PAUSE_PRINT = "Pause print";
+const ACTION_ABORT_PRINT = "Abort print";
+const ACTION_SET_TEMP = "Set temp";
+const ACTION_RUN_MACRO = "Run macro";
```

Comment:
I like having these as strings, though having them be the *same* strings as in the UI could be confusing.  How about having these be `const ACTION_TOGGLE_SCREENSAVER = "TOGGLE_SCREENSAVER"` or something?